### PR TITLE
Fixed container image name

### PR DESCRIPTION
**Pull request by supermaxiste**
Using the current image name on the README doesn't work. The repository name must be lowercase and we have to add the commit on top of the version. We should also update this in Holger's repo.

### CODE REVIEW

**Comment by cmdoret** [comment]
Thanks @supermaxiste I agree we need to make the path lowercase.

However, `ghcr.io/SDSC-ORD/shacl:x.y.z` _will_ work when the release x.y.z is created. It's true it doesn't work for 1.4.2 because we did not retroactively build images. I will add it manually, but for future release, the CI job should create them.

I don't think we need to update the upstream repo, as its readme says to first `build` the image, and then `run` it, it never mentions `docker pull`. I believe these instructions should work.

**Comment by cmdoret** [comment]
`sdsc-ord/shacl:1.4.2` is now available (built on the upstream 1.4.2 commit)

File: README.md [hunk 1]
```diff
@@ -87,26 +87,26 @@ The tools print the validation report or the inferences graph to the output scre
 The `Dockerfile` in the `.docker` folder includes a minimal Java Runtime Environment for the SHACL API that clocks in at 144Mb. To get the latest release of the image use:
 
 ```
-docker pull ghcr.io/SDSC-ORD/shacl:1.4.2
+docker pull ghcr.io/sdsc-ord/shacl:1.4.2_09e93ff
```
Comment (cmdoret) [inline README.md:90]:
```suggestion
docker pull ghcr.io/sdsc-ord/shacl:latest
```

File: README.md [hunk 2]
```diff
@@ -87,26 +87,26 @@ The tools print the validation report or the inferences graph to the output scre
...
 ```
 
 To use the Docker image, there are two possible commands. To run the validator:
 
 ```
-docker run --rm -v /path/to/data:/data ghcr.io/SDSC-ORD/shacl:1.4.2 validate -datafile /data/myfile.ttl -shapesfile /data/myshapes.ttl
+docker run --rm -v /path/to/data:/data ghcr.io/sdsc-ord/shacl:1.4.2_09e93ff validate -datafile /data/myfile.ttl -shapesfile /data/myshapes.ttl
```
Comment (cmdoret) [inline README.md:96]:
```suggestion
docker run --rm -v /path/to/data:/data ghcr.io/sdsc-ord/shacl:latest validate -datafile /data/myfile.ttl -shapesfile /data/myshapes.ttl
```

File: README.md [hunk 3]
```diff
@@ -87,26 +87,26 @@ The tools print the validation report or the inferences graph to the output scre
...
 ```
 
 To run rule inferencing:
 
 ```
-docker run --rm -v /path/to/data:/data ghcr.io/SDSC-ORD/shacl:1.4.2 infer -datafile /data/myfile.ttl -shapesfile /data/myshapes.ttl
+docker run --rm -v /path/to/data:/data ghcr.io/sdsc-ord/shacl:1.4.2_09e93ff infer -datafile /data/myfile.ttl -shapesfile /data/myshapes.ttl
```
Comment (cmdoret) [inline README.md:102]:
```suggestion
docker run --rm -v /path/to/data:/data ghcr.io/sdsc-ord/shacl:latest infer -datafile /data/myfile.ttl -shapesfile /data/myshapes.ttl
```

File: README.md [hunk 4]
```diff
@@ -87,26 +87,26 @@ The tools print the validation report or the inferences graph to the output scre
...
 ```
 
-Any other command after `ghcr.io/SDSC-ORD/shacl:1.4.2` will print the following help page:
+Any other command after `ghcr.io/sdsc-ord/shacl:1.4.2_09e93ff` will print the following help page:
```
Comment (cmdoret) [inline README.md:105]:
```suggestion
Any other command after `ghcr.io/sdsc-ord/shacl:latest` will print the following help page:
```

File: README.md [hunk 5]
```diff
@@ -87,26 +87,26 @@ The tools print the validation report or the inferences graph to the output scre
...
 
 ```
 Please use this docker image as follows:
-docker run -v /path/to/data:/data ghcr.io/SDSC-ORD/shacl:1.4.2 [COMMAND] [PARAMETERS]
+docker run -v /path/to/data:/data ghcr.io/sdsc-ord/shacl:1.4.2_09e93ff [COMMAND] [PARAMETERS]
```
Comment (cmdoret) [inline README.md:109]:
```suggestion
docker run -v /path/to/data:/data ghcr.io/sdsc-ord/shacl:latest [COMMAND] [PARAMETERS]
```

File: README.md [hunk 6]
```diff
@@ -122,5 +122,5 @@ PARAMETERS:
 If you'd like to build the image locally, use:
 
 ```
-docker build -f .docker/Dockerfile -t ghcr.io/SDSC-ORD/shacl:1.4.2 --build-arg VERSION=1.4.2 .
+docker build -f .docker/Dockerfile -t ghcr.io/sdsc-ord/shacl:1.4.2_09e93ff --build-arg VERSION=1.4.2 .
```
Comment (cmdoret) [inline README.md:125]:
```suggestion
docker build -f .docker/Dockerfile -t ghcr.io/sdsc-ord/shacl:latest --build-arg VERSION=1.4.2 .
```

File: README.md [hunk 7]
```diff
@@ -87,26 +87,26 @@ The tools print the validation report or the inferences graph to the output scre
 The `Dockerfile` in the `.docker` folder includes a minimal Java Runtime Environment for the SHACL API that clocks in at 144Mb. To get the latest release of the image use:
 
 ```
-docker pull ghcr.io/SDSC-ORD/shacl:1.4.2
+docker pull ghcr.io/sdsc-ord/shacl:1.4.2_09e93ff
 ```
```
Comment (cmdoret) [inline README.md:92]:
```suggestion
> :warning: It is generally better to used a fixed version of the docker image, rather than the `latest` tag. Consult the [package page](https://github.com/SDSC-ORD/shacl/pkgs/container/shacl) find what versions are available.
```

**Review by cmdoret — CHANGES_REQUESTED** [review]

**Comment by cmdoret** [comment]
tl;dr I suggested to replace all mention of the image tag by `latest` and add a notice pointing to the container page to select a fixed version. Would that make sense?

**Comment by rmfranken** [comment]
Haha, issue made and solved before I even get a chance to see it! Can confirm @cmdoret's fix works: upper-case breaks, lower-case works (kind of a silly error though...) Thanks for the sharp eye @supermaxiste .

Now lets hope we actually get any kind of traffic to justify the time we spent on this, because so far the latest release had only been downloaded 3 times (1 of which is definitely me, the other 2 probably you! 😄

**Review by rmfranken — APPROVED** [review]
